patterns/core/sass/settings/_colors.scss

```diff
@@ -0,0 +1,7 @@
+$ColorScheme: (
```

Comment:
The variable naming scheme and rationale -- just noting to document this in a wiki or some other place for future contribs